lms/templates/certificates/valid.html

```diff
@@ -3,9 +3,9 @@
 <% mako.runtime.UNDEFINED = '' %>
 
 <%inherit file="accomplishment-base.html" />
-
+% if user.is_authenticated():
```

Comment:
@ziafazal -- I'm not sure this is enough of a check -- we need to ensure that the user is authenticated, but also that the user is the same user as the one for the certificate -- after all, it's possible that two authenticated users could view each others' certificates.